src/main/java/edu/umich/tl/Utils.java

```diff
@@ -64,6 +66,13 @@ public static InputStream getInputStreamFromURL(String fileURL) {
 
 		return in;
 	}
+	public static String dateChopper(String dateAsString) {
```

Comment:
It appears that this is trying to parse a date and time in ISO 8601 format.  It may be worthwhile to use an existing parser that can handle that format.  For example, `javax.xml.bind.DatatypeConverter.parseDateTime(dateAsString).getTime()` to get a Date object.  If you really need to use a string, append `.toString()`.